app/helpers/application_helper.rb

```diff
@@ -8,4 +8,27 @@ def to_options(array)
     result.unshift(OpenStruct.new(name: nil))
     result
   end
+
+  def govuk_button_link_to(body, url, html_options = {}, &_block)
```

Comment:
No need for the last argument `&_block` since the method body is using `yield` and `block_given?`. It can be removed

Reply:
thanks, removed